spark/dl/src/main/scala/com/intel/analytics/bigdl/optim/Predictor.scala

```diff
@@ -209,7 +209,11 @@ object Predictor {
         splitBatch(output, shareBuffer, batch.size())
       })
       partition.zip(batchOut).map(sample => {
-        Sample(sample._1.feature(), sample._2.toTensor)
+        if (sample._2.toTensor == null) {
```

Comment:
why this change ? isn't it supposed to have label after prediction ?

Reply:
I think the batchOut are the predicted label array. So I zip the partition with the batchOut.

Reply:
I mean why you add the null checking ?